src/core/main.js

```diff
@@ -763,4 +744,38 @@ p5.registerAddon(renderer);
 p5.registerAddon(renderer2D);
 p5.registerAddon(graphics);
 
-export default p5;
+const p5Proxy = new Proxy(p5, {
+  construct(target, args){
```

Comment:
Ah yeah I see what you mean about the proxy no longer being necessary, we probably could cut and paste this into the regular constructor now. Maybe worth doing to see if tests pass, just to avoid any other possible complications that come from having a proxy

Reply:
Cool, I'll just move it to the very start of the constructor and not use Proxy.